api/src/main/resources/liquibase.xml

```diff
@@ -16,15 +16,19 @@
     xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
 	
-	<changeSet id="data-integrity-201610211035" author="ssmusoke" dbms="mysql">
```

Comment:
Have you looked into the effects of changing the id of an existing changeset?
https://wiki.openmrs.org/display/docs/How+to+Modify+a+Bad+Changeset